cropro.py

```diff
@@ -179,6 +201,22 @@ def initUI(self):
 
         self.setWindowTitle('Cross Profile Search and Import')
         self.exec_()
+    
+    def playAudio(self):
+        if not self.audioCheckBox.isChecked():
+            return
```

Comment:
Method name does not represent what it does. This if statement should be placed outside this method.